src/interpreter.rs

```diff
@@ -21,6 +21,15 @@ use memory::{Memory, Pointer};
 use primval::{self, PrimVal};
 
 const TRACE_EXECUTION: bool = true;
+static mut INDENTATION: usize = 0;
```

Comment:
I have [a branch](https://github.com/oli-obk/miri/commit/a39252a306462a037c1017b35f4b8a368bf49ce8) that gets us `debug!` support from the `log` crate. I'll simply remove this commit from this PR and do everything in the log branch